common/http/src/main/java/io/helidon/common/http/Http1HeadersParser.java

```diff
@@ -58,6 +67,10 @@ public static WritableHeaders<?> readHeaders(DataReader reader, int maxHeadersSi
 
             Http.HeaderName header = readHeaderName(reader, maxLength, validate);
             maxLength -= header.defaultCase().length() + 2;
+            // Skip spaces after header name
```

Comment:
This is not needed - it is already part of `LazyString` - there is a method `stripOws` that handles this problem.
This would only introduce a performance overhead

Reply:
Removed